components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2706,6 +2715,10 @@ public void populateOAuthTokenIssuerMap() throws IdentityOAuth2Exception {
         }
     }
 
+    public boolean isScopeValidationEnabledForCodeAndImplicitGran() {
```

Comment:
missing 't' at the end 
isScopeValidationEnabledForCodeAndImplicitGran -> isScopeValidationEnabledForCodeAndImplicitGrant

Reply:
Let's add Java Doc

Reply:
done